docs/resources/consul_cluster.md

```diff
@@ -38,6 +38,7 @@ resource "hcp_consul_cluster" "example" {
 ### Optional
 
 - **auto_hvn_to_hvn_peering** (Boolean) Enables automatic HVN to HVN peering when creating a secondary cluster in a federation.
+The alternative to using the auto-accept feature is to create an [`hcp_hvn_peering_connection`](hvn_peering_connection.md) resource that explicitly defines the HVN resources that are allowed to communicate with each other.
```

Comment:
This doc is generated, so you'll want to add this line here: https://github.com/hashicorp/terraform-provider-hcp/blob/main/internal/provider/resource_consul_cluster.go#L142
And then run `go generate ./...`

Reply:
Thank you @bcmdarroch. I went ahead and make the changes in the correct spot.